src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -0,0 +1,25 @@
+@echo off
+setlocal
+
+set PROJECT_DIR=%~1
+set DOTNET_CMD=%~2
+set TOOLRUNTIME_DIR=%~3
+set BUILDTOOLS_PACKAGE_DIR=%~dp0
+
+if not exist "%PROJECT_DIR%" (
+  echo ERROR: Cannot find project root path at [%PROJECT_DIR%]. Please pass in the source directory as the 1st parameter.
+  exit /b 1
+)
+
+if not exist "%DOTNET_CMD%" (
+  echo ERROR: Cannot find dotnet cli at [%DOTNET_CMD%]. Please pass in the path to dotnet.exe as the 2nd parameter.
+  exit /b 1
+)
+
+cd %BUILDTOOLS_PACKAGE_DIR%\tool-runtime\
+call %DOTNET_CMD% restore
+call %DOTNET_CMD% publish -f dnxcore50 -r win7-x64 -o %TOOLRUNTIME_DIR%
```

Comment:
There is a good chance you will need to add quotes around these env variables because if anyone has a space in the repo path it will fail otherwise.